### PR TITLE
Processor function documentation and minor fixes

### PR DESCRIPTION
**Pull request by tompaana**
# PR for issue #415 

## What is being addressed

Documents processor function and adds minor improvements and clean up.

### CODE REVIEW

File: processor_function/README.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+The processor function (sometimes referred to as resource or deployment processor) uses [Azure Container Instances (ACI)](https://docs.microsoft.com/en-us/azure/container-instances/) together with the [CNAB container image](../CNAB_container/Dockerfile) based on [Azure CNAB Driver](https://github.com/deislabs/cnab-azure-driver) to execute workspace and workspace service deployments. The workspaces and workspace services are [Porter](https://porter.sh/) bundles.
+
+<!-- markdownlint-disable-next-line MD013 -->
+The processor function waits for Service Bus messages in the resource request queue containing the bundle details and the action to execute, prepares the environment for the execution in the ACI and follows through the deployment process. The deployment status is reported back to [the management API](../management_api_app/) using Service Bus messages but with in a deployment status update queue.
+
+## Prerequisites
+
+Tools:
+
+* [Python 3.x](https://www.python.org/downloads/)
```
Comment (Lybecker) [inline processor_function/README.md:13]: @sachinkundu I think we are using a specific version, right?

File: processor_function/README.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Processor function
+
+<!-- markdownlint-disable-next-line MD013 -->
+The processor function (sometimes referred to as resource or deployment processor) uses [Azure Container Instances (ACI)](https://docs.microsoft.com/en-us/azure/container-instances/) together with the [CNAB container image](../CNAB_container/Dockerfile) based on [Azure CNAB Driver](https://github.com/deislabs/cnab-azure-driver) to execute workspace and workspace service deployments. The workspaces and workspace services are [Porter](https://porter.sh/) bundles.
```
Comment (Lybecker) [inline processor_function/README.md:4]: perhaps mention something about the responsibility

**Review by Lybecker — APPROVED** [review]
LGTM - a few comments

File: processor_function/README.md (hunk 1 above)
Comment (tompaana) [reply processor_function/README.md:13]: The dev container has 3.8. Updating the README accordingly.

File: processor_function/README.md (hunk 2 above)
Comment (tompaana) [reply processor_function/README.md]: Added some more information. We can revisit this in #431

**Pull request merged**